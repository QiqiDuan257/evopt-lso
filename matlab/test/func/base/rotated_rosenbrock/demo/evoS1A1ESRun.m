clear; clc;

% nohup matlab -nodesktop -nodisplay -nosplash <evoS1A1ESRun.m >evoS1A1ESRun.out 2>&1 &

startRun = tic;
rng(20190101); % for reproducibility
funcParam.funcName = 'evoRF_Rosenbrock';
% funcParam.funcDims = [2 10];
algoParam.algoName = 'evoS1A1ES';
algoParam.algoStopt = 3600 * 4;
[statOptRes, funcParams, algoParams] = ...
    evoCreateFAEBaseFunc(funcParam, algoParam);
fprintf(sprintf('$-runtime-$: %7.5e.\n', toc(startRun)));
