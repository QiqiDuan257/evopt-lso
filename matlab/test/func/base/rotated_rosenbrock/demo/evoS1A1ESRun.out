*** funcDim: 2 + algoName: evoS1A1ES + maxFEval: 200000 ***
    opty = +4.4532e-02 | runtime = 1.32e+01 | timeFEval = 7.05e+00 | numFEval = 200000 | timeCcFEval = 6.16e-01 | stopFlag = maxFEval
*** funcDim: 10 + algoName: evoS1A1ES + maxFEval: 1000000 ***
    opty = +9.9970e-11 | runtime = 2.04e+01 | timeFEval = 1.08e+01 | numFEval = 376585 | timeCcFEval = 1.03e+00 | stopFlag = stopy
*** funcDim: 100 + algoName: evoS1A1ES + maxFEval: 10000000 ***
    opty = +9.9998e-11 | runtime = 4.42e+02 | timeFEval = 2.53e+02 | numFEval = 6480104 | timeCcFEval = 1.86e+01 | stopFlag = stopy
*** funcDim: 1000 + algoName: evoS1A1ES + maxFEval: 100000000 ***
    opty = +3.9866e+00 | runtime = 1.43e+04 | timeFEval = 8.47e+03 | numFEval = 100000000 | timeCcFEval = 3.36e+02 | stopFlag = maxFEval
*** funcDim: 2000 + algoName: evoS1A1ES + maxFEval: 200000000 ***
    opty = +8.7536e+02 | runtime = 1.44e+04 | timeFEval = 1.04e+04 | numFEval = 56216062 | timeCcFEval = 2.05e+02 | stopFlag = stopt
*** funcDim: 4000 + algoName: evoS1A1ES + maxFEval: 400000000 ***
    opty = +4.3386e+03 | runtime = 1.44e+04 | timeFEval = 1.38e+04 | numFEval = 2759835 | timeCcFEval = 2.78e+01 | stopFlag = stopt
*** funcDim: 6000 + algoName: evoS1A1ES + maxFEval: 600000000 ***
    opty = +6.6440e+03 | runtime = 1.44e+04 | timeFEval = 1.40e+04 | numFEval = 1445526 | timeCcFEval = 1.48e+01 | stopFlag = stopt
*** funcDim: 8000 + algoName: evoS1A1ES + maxFEval: 800000000 ***
    opty = +9.7530e+03 | runtime = 1.44e+04 | timeFEval = 1.41e+04 | numFEval = 880867 | timeCcFEval = 9.17e+00 | stopFlag = stopt
*** funcDim: 10000 + algoName: evoS1A1ES + maxFEval: 1000000000 ***
    opty = +2.2626e+04 | runtime = 1.44e+04 | timeFEval = 1.42e+04 | numFEval = 599070 | timeCcFEval = 7.25e+00 | stopFlag = stopt
$-runtime-$: 8.67980e+04.
